packages/settingeditor-extension/src/index.ts

```diff
@@ -45,6 +50,88 @@ namespace CommandIDs {
  * The default setting editor extension.
  */
 const plugin: JupyterFrontEndPlugin<ISettingEditorTracker> = {
+  id: '@jupyterlab/settingeditor-extension:simple-plugin',
+  requires: [
+    ILayoutRestorer,
+    ISettingRegistry,
+    IStateDB,
+    ITranslator,
+    IFormComponentRegistry
+  ],
+  optional: [ICommandPalette],
+  autoStart: true,
+  provides: ISettingEditorTracker,
+  activate
+};
+
+/**
+ * Activate the setting editor extension.
+ */
+function activate(
+  app: JupyterFrontEnd,
+  restorer: ILayoutRestorer,
+  registry: ISettingRegistry,
+  state: IStateDB,
+  translator: ITranslator,
+  editorRegistry: IFormComponentRegistry,
+  palette?: ICommandPalette
+): ISettingEditorTracker {
+  const trans = translator.load('jupyterlab');
+  const { commands, shell } = app;
+  const namespace = 'setting-editor';
+  const tracker = new WidgetTracker<MainAreaWidget<SimpleSettingsEditor>>({
+    namespace
+  });
+
+  // Handle state restoration.
+  void restorer.restore(tracker, {
+    command: CommandIDs.open,
+    args: widget => ({}),
+    name: widget => namespace
+  });
+
+  if (palette) {
+    palette.addItem({
+      category: trans.__('Simple Settings'),
+      command: CommandIDs.open
+    });
+  }
+
+  commands.addCommand(CommandIDs.open, {
+    execute: () => {
+      if (tracker.currentWidget) {
+        shell.activateById(tracker.currentWidget.id);
+        return;
+      }
+
+      const key = plugin.id;
+
+      const editor = new SimpleSettingsEditor({
+        editorRegistry,
+        key,
+        registry,
+        state,
+        translator
+      });
+
+      editor.id = namespace;
+      editor.title.icon = settingsIcon;
+      editor.title.label = trans.__('Settings');
+
+      const main = new MainAreaWidget({ content: editor });
+      void tracker.add(main);
+      shell.add(main);
+    },
+    label: trans.__('Simple Settings Editor')
```

Comment:
Let's call it `Settings Editor` as this is certainly not simple :wink: 

```suggestion
    label: trans.__('Settings Editor')
```